moderator/settings.py

```diff
@@ -122,7 +121,7 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 
 HMAC_KEYS = config('HMAC_KEYS', cast=json.loads)
 
-PASSWORD_HASHERS = get_password_hashers(BASE_PASSWORD_HASHERS, HMAC_KEYS)
+PASSWORD_HASHERS = get_password_hashers(PASSWORD_HASHERS, HMAC_KEYS)
```

Comment:
We need to keep these settings in order to be able to upgrade.
https://docs.djangoproject.com/en/1.8/topics/auth/passwords/#password-upgrading